src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461;$(NetFrameworkCurrent)</TargetFrameworks>
```

Comment:
Did you need to change this since it isn't in the package?

Reply:
Not really, initially it was in the package so I changed to add the config, but then when building the package I then hit the error of not being able to have ExcludeCompile on desktop, so then I had to remove the ref from the package but didn't undo this change. I'm happy to undo this if you want me to.

Reply:
I don't feel too strongly, but it seemed like an unnecessary change.

Reply:
yeah, the only reasons I left it were because I had already done it, it wouldn't hurt to have extra configs, and the potential case we ever want to add the refs back to the package, which I guess in that case the test would catch it. I agree with you it is unnecessary so I'll remove them.